ctapipe/image/reducer.py

```diff
@@ -42,5 +116,68 @@ class NullDataVolumeReducer(DataVolumeReducer):
     Perform no data volume reduction
     """
 
-    def __call__(self, waveforms):
-        return waveforms
+    def select_pixels(self, waveforms, telid=None, selected_gain_channel=None):
+        mask = waveforms != 0
+        return mask
+
+
+class TailCutsDataVolumeReducer(DataVolumeReducer):
```

Comment:
This shoudl probably just have a child component `TailCutsImageCleaner` like this, you'd inherit the configuration options automatically.

Reply:
Alright, added the `TailCutsImageCleaner` as a subcomponent.